src/crimson/common/errorator.h

```diff
@@ -374,6 +382,16 @@ struct errorator {
     template <class T>
     using get_errorator_t = typename get_errorator<T>::type;
 
+    template <typename T>
+    struct futurized_error {
+      using ErrorT = T;
+    };
+
+    template <typename T>
+    struct futurized_error<seastar::future<T>> {
+      using ErrorT = T;
```

Comment:
What's the purpose of `futurized_error`? At first glance it looks somewhere else (futurator?) is an issue causing to wrap our error in future (`seastar::future<crimson::ct_error::something>`).

Reply:
Sorry, this shouldn't exist anymore, I think I forgot to remove it